integration/test-integration/pom.xml

```diff
@@ -20,7 +20,7 @@
     <parent>
         <groupId>org.wso2.choreo.connect</groupId>
         <artifactId>integration</artifactId>
-        <version>1.1.0-beta-SNAPSHOT</version>
```

Comment:
Please check where there are multiple places need to change in this file. Previous PRs has two places.